types/mysql/index.d.ts

```diff
@@ -1,177 +1,274 @@
-// Type definitions for node-mysql
-// Project: https://github.com/felixge/node-mysql
+// Type definitions for mysql 2.15.0
+// Project: https://github.com/mysqljs/mysql
 // Definitions by: William Johnston <https://github.com/wjohnsto>
 //		   Kacper Polak <https://github.com/kacepe>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
 ///<reference types="node" />
 
 
 import stream = require("stream");
+import tls = require("tls");
+
+interface IEscape {
+    escape(value: any, stringifyObjects?: boolean, timeZone?: string): string;
+
+    escapeId(value: string, forbidQualified?: boolean): string;
+
+    format(sql: string, values: any[], stringifyObjects?: boolean, timeZone?: string): string;
+}
+
+/**
+ * @deprecated
+ */
+
+interface IMySql {
```

Comment:
I would just remove this This PR is a major version change since you updated the header.

Reply:
it used by express-myconnection

Reply:
That package should probably be updated then? Because a function accepting `IMySql` would now accept almost anything, since it's an empty interface.

Reply:
Update of this package should be in this pr or separated?

Reply:
Together, because this PR effectively changes the meaning of the function in `express-myconnection`.